.github/workflows/code_coverage.yml

```diff
@@ -0,0 +1,98 @@
+name: Code Coverage
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    paths-ignore:
+      - '.github/workflows/**'
+      - 'docker/**'
+      - 'doc/**'
+      - 'CHANGELOG.rst'
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ubuntu_versions : [
+          20.04,
+          22.04,
+        ]
+        pkg_mgr : [
+          apt,
+          conda,
+        ]
+    
+    container:
+      image: ghcr.io/cyclus/cyclus_${{ matrix.ubuntu_versions }}_${{ matrix.pkg_mgr }}/${{ matrix.pkg_mgr }}-deps
```

Comment:
do we still maintain a `-deps` image with this workflow?

Reply:
No we do not.  Working on a solution now